.dockerignore

```diff
@@ -1,11 +1,14 @@
 # Blanket ignore everything by default
 *
+cdk*
 
 # Whitelist files and folders that should be in the container
 !app
 !migrations
 !scripts
 
+!phuks.txt
+!config.yaml
```

Comment:
It was pointed out to me that this might allow somebody to do something stupid and publish a file full of secrets to dockerhub.

We don't know the right way to handle this, but it is a potential vulnerability

Reply:
https://github.com/aws/containers-roadmap/issues/56 some different people talking about similar issues